setup.cfg

```diff
@@ -50,8 +50,8 @@ filterwarnings=
     ignore::pytest.PytestUnraisableExceptionWarning
     ignore:numpy.ndarray size changed:RuntimeWarning
     # ignore benign Cython warnings on ndarray size
-    ignore::DeprecationWarning:ml_dtypes.*
-    # ignore benign ml_dtypes DeprecationWarning
+    ignore::DeprecationWarning:(?!desc)
+    # ignore DeprecationWarnings from dependency packages
```

Comment:
On this occasion, can you fix the comment order here? The comment being on top the line it explains is better imo

Reply:
Or wait tests are almost done. We merge this in, and then fix that

Reply:
I agree that as a standard we should always put comments on the line above, not below, the code it is related to. Let's just fix it now because opening a whole new PR will take longer.